docs/pagination.md

```diff
@@ -0,0 +1,71 @@
+# Pagination
+
+Pagination is supported using the metadata support that is built into Ember Data. The
+metadata from Django REST Framework paginated list views is updated on every request
+to sever.
+
+
+## Retrieving the Metadata
+
+To get a page of records simply run a find request with the `page` query param:
+
+```js
+var result = this.store.find("post", {
+  page: 1
+});
+```
+
+It's possible to override the server-side page size by also including the `page_size`
```

Comment:
It turns out there are no pagination related defaults in DRF:
https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/settings.py#L61

I usually set `paginated_by` on the view to enable pagination and I usually use different page sizes across my paginated views  (i.e. I don't use the `PAGINATE_BY` setting). This is one reason why I'm pushing for setting the `pageSize` in the metadata.

I'm going to add the `pageSize` to the metadata for you to review. I'm pushing hard for this point because I think it makes things easier for application developers. Do you think having the `pageSize` in the metadata is a bad idea?